features/lockfile_spec.rb

```diff
@@ -0,0 +1,17 @@
+require File.expand_path('../support/command_rspec_helper', __FILE__)
+
+describe 'rmt-cli' do
+  before(:example) { system '/usr/bin/rmt-cli sync > /dev/null &' }
+  # kill running process to let further specs pass
+  after(:example) { system 'kill -9 $(cat /tmp/rmt.lock)' }
```

Comment:
the default scope of `before` and `after` hooks is `:example` (or `:each`, which is the same) so you don't need to specify it

Reply:
fixed